backend/apps/slack/apps.py

```
@@ -24,13 +24,30 @@
 if SlackConfig.app:
 
     @SlackConfig.app.error
-    def error_handler(error, body, *args, **kwargs):  # noqa: ARG001
-        """Handle Slack application errors."""
+    def error_handler(error, body, *args, **kwargs):
```

Ruff (0.8.2)

Unused function argument: `args`

(ARG001)